### PR TITLE
start kafka container for kafka and streams tests

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
GH actions failed because of this ```[INFO] --- maven-checkstyle-plugin:3.1.1:check (validate) @ apicurio-registry-storage-streams ---
[INFO] Starting audit...
[ERROR] /home/runner/work/apicurio-registry/apicurio-registry/storage/streams/target/generated-sources/protobuf/grpc-java/io/apicurio/registry/streams/distore/proto/KeyValueStoreGrpc.java:4:15: Unused import - io.grpc.stub.ClientCalls.asyncBidiStreamingCall. [UnusedImports]```

**Comment by famartinrh** [comment]
Now the GH actions run show the error I'm after for a while,
```
[INFO] Running io.apicurio.registry.ConfluentClientTest
2020-05-24 18:09:12,416 INFO  [org.apa.htt.imp.cli.DefaultHttpClient] (main) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://localhost:8081: The target server failed to respond
2020-05-24 18:09:12,416 INFO  [org.apa.htt.imp.cli.DefaultHttpClient] (main) Retrying request to {}->http://localhost:8081
```
The registry API is "diying" and stops responding after some load, it only happens with streams and kafka storage (focus investigation only in streams storage please). I don't have more details of in what exact circunstances it happens, but I need help investigating this issue and finding a fix. At least now we have this PR so everyone can easily reproduce the error. Thanks

File: storage/kafka/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.KafkaContainer;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class KafkaTestContainerManager implements QuarkusTestResourceLifecycleManager {
```
Comment (alesj) [inline storage/kafka/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java:25]: You could put this class -- so it's not a dup -- probably into util/kafka under test/ sources.

Comment (alesj) [reply storage/kafka/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java:25]: > The registry API is "diying" and stops responding after some load

What if you run the tests with the streams_setup.sh script?
That's how I do it, and they worked with np so far ...

Comment (EricWittmann) [reply storage/kafka/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java:25]: @alesj can you work with @famartinrh to try and track down the issue of the server becoming unresponsive when using the streams storage?  A previous run appeared to have logging information indicating that connections between registry and kafka were failing... 

Here is a link to that log info:  https://bit.ly/2yuQbW4

**Comment by alesj** [comment]
Fixed by #532

**Pull request closed without merging**